docs/appendix/zowe-glossary.md

```diff
@@ -197,7 +197,7 @@ The Zowe Support Provider Conformance Program gives vendors the ability to showc
 
 #### Base profile
 
-An object in a team configuration file that stores connection information for use with one or more services. Your service profiles can pull information from base profiles as needed, to specify a common username and password only once.
+An object in a team configuration file that stores connection information for use with one or more services. Depending on your configuration file type, the base profile can be either a `global_base` or `project_base` profile. Your service profiles can pull information from base profiles as needed, to specify a common username and password only once.
```

Comment:
The new sentence looks good. But I think it would be clearer to change the last sentence in this paragraph to "Your service profiles can pull information from base profiles as needed, to specify a common username and password only once, **for example**."